src/main/java/org/broadinstitute/hellbender/engine/spark/SparkCommandLineArgumentCollection.java

```diff
@@ -17,17 +18,29 @@
 public final class SparkCommandLineArgumentCollection implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    @Argument(fullName = "spark-master", doc="URL of the Spark Master to submit jobs to when using the Spark pipeline runner.", optional = true)
+    public static final String SPARK_MASTER_LONG_NAME = "spark-master";
+    public static final String SPARK_VERBOSITY_LONG_NAME = "spark-verbosity";
+
+    @Argument(
+            doc="URL of the Spark Master to submit jobs to when using the Spark pipeline runner.",
+            fullName = SPARK_MASTER_LONG_NAME,
+            optional = true)
     private String sparkMaster = SparkContextFactory.DEFAULT_SPARK_MASTER;
 
     @Argument(
-            doc = "spark properties to set on the spark context in the format <property>=<value>",
-            shortName = StandardArgumentDefinitions.SPARK_PROPERTY_NAME,
+            doc = "Spark properties to set on the Spark context in the format <property>=<value>",
             fullName = StandardArgumentDefinitions.SPARK_PROPERTY_NAME,
+            shortName = StandardArgumentDefinitions.SPARK_PROPERTY_NAME,
             optional = true
     )
     final List<String> sparkProperties = new ArrayList<>();
 
+    @Argument(
+            doc="Spark verbosity (ALL, DEBUG, INFO, WARN, ERROR, FATAL, OFF, TRACE)",
+            fullName = SPARK_VERBOSITY_LONG_NAME,
+            optional = true)
+    private String sparkVerbosity = Level.WARN.name(); // Default INFO is too verbose
```

Comment:
Hmm... I'm torn about making this the default. I suspect that many long running tools on the command line via spark might be confusingly silent to users who may think the program has hung. For example I just tried to run MarkDuplicatesSpark and saw no output for several minutes at a time. Perhaps this is bearable? Ultimately the issue is that we can't have a read/variant counter in spark like we have built into the walkers. As a result I suggest not changing the default here. 

If we do keep the default we could do one of two things to change the behavior for problematic tools. First we could make it tool-level configurable by adding a `getToolDefaultSparkVerbosity()` method that sets this at the GATKSparkToolLevel that defaults to INFO and just set WARN for specific tools that are causing problems. Alternatively and possibly as a better option we can just change the wlds that that are problems.

Reply:
As a corollary to that we should check the travis log output and make sure its not innundated with unwanted spark outputs.